packages/svelte/src/use-chat.ts

```diff
@@ -101,19 +132,14 @@ const getStreamedResponse = async (
           name,
           data,
           annotations,
-          function_call,
-          tool_calls,
-          tool_call_id,
+          toolInvocations,
         }) => ({
           role,
           content,
           ...(name !== undefined && { name }),
           ...(data !== undefined && { data }),
           ...(annotations !== undefined && { annotations }),
-          // outdated function/tool call handling (TODO deprecate):
-          tool_call_id,
-          ...(function_call !== undefined && { function_call }),
-          ...(tool_calls !== undefined && { tool_calls }),
```

Comment:
please keep for now, same above/below. the old behavior is deprecated but existing users might rely on it.